### ISSUE TITLE
Scanning File Error: 'MPEGInfo' object has no attribute '_MPEGInfo__BITRATE'

### ISSUE THREAD

**Issue by starenka** [Nicotine-Plus/nicotine-plus#51]
```
08:08:12 Scanning File Error: 'MPEGInfo' object has no attribute '_MPEGInfo__BITRATE' Path: ***
08:08:12 Traceback: AttributeError: 'MPEGInfo' object has no attribute '_MPEGInfo__BITRATE'
08:08:12   File "/usr/lib/python2.7/dist-packages/pynicotine/shares.py", line 568, in getFileInfo
    info = metadata.detect(pathname)
08:08:12   File "/usr/lib/python2.7/dist-packages/pynicotine/metadata_mutagen.py", line 26, in detect
    return processMPEG(audio)
08:08:12   File "/usr/lib/python2.7/dist-packages/pynicotine/metadata_mutagen.py", line 62, in processMPEG
    rates = audio.info._MPEGInfo__BITRATE[(audio.info.version, audio.info.layer)]
08:08:12 Traceback: AttributeError: 'MPEGInfo' object has no attribute '_MPEGInfo__BITRATE'
08:08:12   File "/usr/lib/python2.7/dist-packages/pynicotine/shares.py", line 568, in getFileInfo
    info = metadata.detect(pathname)
08:08:12   File "/usr/lib/python2.7/dist-packages/pynicotine/metadata_mutagen.py", line 26, in detect
    return processMPEG(audio)
08:08:12   File "/usr/lib/python2.7/dist-packages/pynicotine/metadata_mutagen.py", line 62, in processMPEG
    rates = audio.info._MPEGInfo__BITRATE[(audio.info.version, audio.info.layer)]
08:08:12 Scanning File Error: 'MPEGInfo' object has no attribute '_MPEGInfo__BITRATE'
```

Debian testing
Package: nicotine                        
Version: 1.2.16+dfsg-1.1

**Comment by gfarmerfr** [comment]
Hi @starenka ,

Sorry for the late response, I was very busy.

This particular problem has been fixed in Nicotine+ 1.4.0 via this commit https://github.com/Nicotine-Plus/nicotine-plus/commit/98f3dfa285656763e6c095ae22a6ae61ee98b764.

This behaviour change was introduced in mutagen 1.32.

**Comment by gfarmerfr** [comment]
Closing this issue since it was fixed in 1.4.0
